src/components/input/types.ts

```diff
@@ -16,3 +16,7 @@ export type StyledInputGroupProps = {
   noBorder?: boolean;
   icon?: SVGIconProps["name"];
 };
+
+export type InputPasswordProps = {
+  showPasswordOnInit: boolean;
```

Comment:
What's the use case of showing password immediately? Seems like a bad security practice

Reply:
nvm, seems like it's a legit use case for ux ;d
https://www.dreamerux.com/articles/lfbj8p97t85ssbne8g7bdjlehmzcp5

Reply:
yeah I didnt have a usecase in mind, but it felt like it would be nice to have this ability